tokenizers/src/tokenizer/encoding.rs

```diff
@@ -110,11 +110,7 @@ impl Encoding {
 
     /// Return the number of sequences combined in this Encoding
     pub fn n_sequences(&self) -> usize {
-        if self.sequence_ranges.is_empty() {
-            1
-        } else {
-            self.sequence_ranges.len()
-        }
+        self.sequence_ranges.len().max(1)
```

Comment:
Same, I would rather not include this in this PR.

The `Whatever` -> `Self` is a pretty big change that touches most of the code base.
We should keep everything that might have other side effects OUT so that it's easier to find the problem later.

On this particular one, I am not particularly fond of the final result, `std::cmp::max(1, self.sequences_ranges.len())` feels much more natural (should we make a change).

Reply:
Done.